src/lib/forms-schema/index.ts

```diff
@@ -156,6 +157,13 @@ const SubmissionEventsSchema = Joi.object().keys({
           .allow(null),
         encryptPdf: Joi.boolean().default(false),
       }),
+    })
+    .when('type', {
+      is: 'SCHEDULING',
+      then: Joi.object().keys({
+        nylasAccountId: Joi.string().required(),
+        nylasSchedulingPageId: Joi.number().required(),
```

Comment:
Is this enough to get the url to send the user to? Are we better to store the url here?